src/omnisharp/protocol.ts

```diff
@@ -284,7 +284,6 @@ export interface AutoCompleteResponse {
 
 export interface ProjectInformationResponse {
     MsBuildProject: MSBuildProject;
-    DotNetProject: DotNetProject;
```

Comment:
Omnisharp no longer has a project information provider for `project.json` projects, so anything that used this was not used.

Reply:
Thanks for removing!